build-tools/packages/bundle-size-tools/package.json

```diff
@@ -25,18 +25,20 @@
 		"build:compile": "fluid-build --task compile",
 		"build:copy": "copyfiles -u 1 \"src/**/*.fsl\" dist",
 		"build:docs": "api-extractor run --local",
+		"build:test": "tsc --project ./src/test/tsconfig.json",
 		"check:biome": "biome check .",
 		"check:format": "npm run check:biome",
 		"ci:build:docs": "api-extractor run",
-		"clean": "rimraf --glob dist \"*.tsbuildinfo\" _api-extractor-temp \"*.build.log\"",
+		"clean": "rimraf --glob dist lib \"*.tsbuildinfo\" _api-extractor-temp \"*.build.log\" nyc",
 		"compile": "fluid-build . --task compile",
 		"eslint": "eslint --format stylish src",
 		"eslint:fix": "eslint --format stylish src --fix",
 		"format": "npm run format:biome",
 		"format:biome": "biome check --write .",
 		"lint": "npm run eslint",
 		"lint:fix": "npm run eslint:fix",
-		"test": "echo \"Error: no test specified\" && exit 1",
+		"test": "npm run test:mocha",
+		"test:mocha": "mocha --forbid-only \"lib/test/**/*.test.js\"",
```

Comment:
Other packages in build tools have a `"test:coverage": "c8 npm run test"` script, seems worth having here too if we're adding the `c8` dep.